Header header

float32[] setpoint
float32[] control_output
float32[] measured_output
